ts/screens/onboarding/FingerprintScreen.tsx

```diff
@@ -0,0 +1,119 @@
+import { Button, Content, Text, View } from "native-base";
+import * as React from "react";
+
+import { NavigationScreenProps } from "react-navigation";
+
+import { connect } from "react-redux";
+
+import AbortOnboardingModal from "../../components/AbortOnboardingModal";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+
+import I18n from "../../i18n";
+import {
+  abortOnboarding,
+  fingerprintAcknowledge
+} from "../../store/actions/onboarding";
+
+import { BiometrySimpleType } from "../../sagas/startup/checkSupportedFingerprintSaga";
+import { Dispatch } from "../../store/actions/types";
+
+type FingerprintScreenNavigationParams = {
+  biometryType: BiometrySimpleType;
+};
+
+type OwnProps = NavigationScreenProps<FingerprintScreenNavigationParams>;
+
+type Props = OwnProps & ReturnType<typeof mapDispatchToProps>;
+
+type State = {
+  showAbortOnboardingModal: boolean;
+  biometryType?: BiometrySimpleType;
```

Comment:
when is `biometryType` undefined?

Reply:
That was due to how code was organized. This variable no longer exists in the component state.